pontoon/base/static/js/locale_manage.js

```diff
@@ -0,0 +1,65 @@
+$(function() {
+
+  // Before submitting the form, update translators and managers
+  $('form').submit(function (e) {
+    $.each(['translators', 'managers'], function(i, value) {
+      var data = $('.user.' + value + ' li').map(function() {
+        return $(this).data('id');
+      }).get();
+      $('[name="' + this + '"]').val(data);
+    });
+  });
+
+  // Switch available users
+  $('.user.available label a').click(function(e) {
+    e.preventDefault();
+
+    $(this).addClass('active').siblings('a').removeClass('active');
+    $('.user.available li').show();
+
+    if ($(this).is('.contributors')) {
+      $('.user.available li:not(".contributor")').hide();
+    }
+
+    $('#available').trigger("keyup").focus();
+  });
+
+  // While in contributors tab, search contributors only
+  $('.menu input[type=search]').on('keyup.search', function(e) {
+    if ($('.user.available label a.contributors').is('.active')) {
+      $('.user.available li:not(".contributor")').hide();
+    }
+  });
+
+  // Set translators arrow direction
+  function setArrow(element, event) {
```

Comment:
This works and is neato, but I would've just put two spans inside and had them each take up half of the list item. Which might be worse for long lists of users, maybe.